test/integration/controllers/category.spec.js

```diff
@@ -29,6 +31,12 @@ describe('Category controller', () => {
     const categoriesResponse = await app.get(endpointUrl).set('Authorization', `Bearer ${accessToken}`)
 
     categoryData._id = categoriesResponse.body[0]._id
+
+    subjectBody.category = categoryData._id
+
+    testSubject = await app.post('/subjects/').set('Authorization', `Bearer ${accessToken}`).send(subjectBody)
+
+    testSubject._id = testSubject.body._id
```

Comment:
shouldn't it be subjectBody._id, not the testSubject?

Reply:
Fixed.